src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ObjectInitializerCompletionProviderTests.cs

```diff
@@ -185,6 +185,43 @@ void goo()
             await VerifyExclusiveAsync(markup, true);
         }
 
+        [Fact]
+        public async Task FieldAndProperty2()
+        {
+            var markup = """
+                public static class TestClass
+                {
+                    private static SimpleRange[] _ranges;
+                
+                    public static void Method()
+                    {
+                        _ranges =
+                        [
+                            new()
+                            {
+                                Start = 1,
+                                End = 3,
+                            },
+                            new()
+                            {
+                                $$
+                            },
+                        ];
+                    }
+                }
+                
+                public struct SimpleRange
+                {
+                    public int Start;
+                    public int End { get; set; }
+                };
+                """;
+
+            await VerifyItemExistsAsync(markup, "Start");
+            await VerifyItemExistsAsync(markup, "End");
+            await VerifyExclusiveAsync(markup, true);
```

Comment:
in general, we would prfer that completion tests go into CompletionCommandHandlerTests.  The provider tests are very broken (Since things can appear in them, and not show up in actual completion).

Reply:
think of thse as legacy tests we need to port over, but have no time to do so :)